packages/kit/src/runtime/client/types.d.ts

```diff
@@ -23,7 +23,7 @@ export type NavigationResult = {
 
 export type BranchNode = {
 	module: CSRComponent;
-	loaded: LoadOutput;
+	loaded: NormalizedLoadOutput;
```

Comment:
If `NormalizedLoadOutput` is exported from the public types I wouldn't consider it an internal type and therefore move it into a public type, possibly `page`.

Reply:
where is it being exported from the public types?

Reply:
Ok don't mind me, I'm stupid. I misread the file name and thought this was one of the public d.ts files.